src/main/java/model/Car.java

```diff
@@ -0,0 +1,125 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Arrays.*;
+import static java.util.stream.Collectors.toList;
+
+public class Car {
+    private int score;
+
+    private final String name;
+
+    public static final int START_SCORE = 1;
```

Comment:
```suggestion
    public static final int START_SCORE = 1;
    
    private int score;

    private final String name;
```
일반적으로 상수로 정의되는 변수는 최상위에 선언하곤합니다 😄

Reply:
상수 위치 변경했습니다~!

1c4c728ef50e37c18f5a865fba3b292711425ce4